src/core/components/hooks/DistrictMap.js

```diff
@@ -70,19 +53,38 @@ const useMap = (location, layer, baseAPIUrl, defaultOptions = {}) => {
         <div style="font-size:1.6rem;padding-bottom:5px;font-weight:700;text-align:center;text-transform:capitalize;">
           ${name}
         </div>
-        <p>Coordinates: ${coordinates}</p>
-        <p>Ownership: ${ownership}</p>
-        <p>Parish: ${parish || 'No parish data'}</p>
+        <p>Number of boys: ${enrollmentData.boys}</p>
+        <p>Number of girls: ${enrollmentData.girls}</p>
+        <p>Total pupils: ${enrollmentData.boys !== 'No Data' && enrollmentData.girls !== 'No Data' ? parseInt(enrollmentData.boys, 10) + parseInt(enrollmentData.girls, 10) : 'No Data'}</p>
```

Comment:
You may want to redo this as well after knowing where we get the configs from. We are trying to avoid hard-coding data sources, as they can change. Let the user decide which json key(s) refer to which data.

Reply:
I see `boys` and `girls` is still hard-coded here. Yet, we want to have flexibility to change education for health. Make changes here too.

Reply:
Updated